cnames_active.js

```diff
@@ -702,6 +702,7 @@ var cnames_active = {
   "dplayer": "diygod.github.io/DPlayer", // noCF
   "dpm": "martondev.github.io/dpm",
   "dprof": "andreasmadsen.github.io/dprof",
+  "Dragonizedpizza": "http://dragonizedpizza.com/Dragonizedpizza.js.org/",
```

Comment:
I'm guessing this is meant to target https://github.com/Dragonizedpizza/Dragonizedpizza.js.org ?

If so, your entry should be formatted like so:
```suggestion
  "dragonizedpizza": "dragonizedpizza.github.io/Dragonizedpizza.js.org",
```

We cannot CNAME target a directory of a random website, it is not possible.

Reply:
@MattIPv4 , I tried to do that, but it kept going to my domain.